_pages/PPDM/Note-on-manufacture-statement.md

```diff
@@ -22,40 +22,35 @@ sidebar:
+<a name="4.226.3.2">4.226.3.2</a> Make a note on manufacture statement if information is not transcribed in the [Manufacture statement](/DCRMR/ppdm/Manufacture-statement/) if considered important (see [4.01.3](/DCRMR/ppdm/#4.01.3)).
 
 <a name="4.226.3.3">4.226.3.3</a>  Make a note on manufacture statement if [Manufacture statements](/DCRMR/ppdm/Manufacture-statement/) are transposed indicating the original position of the transposed elements (see [4.02.2](/DCRMR/ppdm/#4.02.2)).
 
-<a name="4.226.3.4">4.226.3.4</a> Make an explanatory note if all information relating to the [Manufacture statement](/DCRMR/ppdm/Manufacture-statement/) appearing in the manifestation is known to be fictitious or incorrect and include the source of this information (see [4.22.3.1](/DCRMR/ppdm/Manufacture-statement/#4.22.3.1)).
-
-<a name="4.226.3.5">4.226.3.5</a> Make a note if any of the original details relating to the [Manufacture statement](/DCRMR/ppdm/Manufacture-statement/) are covered by a label, stamp, overprint, or other means showing later information and the original details are visible or otherwise available (see [4.22.4.1](/DCRMR/ppdm/Manufacture-statement/#4.22.4.1)). 
+<a name="4.226.3.4">4.226.3.4</a>  Make a note if any of the original details relating to the [Manufacture statement](/DCRMR/ppdm/Manufacture-statement/) are covered by a label, stamp, overprint, or other means showing later information and the original details are visible or otherwise available (see [4.22.4.1](/DCRMR/ppdm/Manufacture-statement/#4.22.4.1)). 
 
 ## 4.226.4 Notes from Place of manufacture
 
-<a name="4.226.4.1">4.226.4.1</a> Make a note on manufacture statement if necessary to justify a supplied [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) (see [4.221.4.1](/DCRMR/ppdm/Place-of-manufacture/#4.221.4.1)).
+<a name="4.226.4.1">4.226.4.1</a> Make a note if the manifestation is issued in more than one physical part, and the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) changes in the course of publication by giving the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) of the later part(s) (see [4.221.9.1](/DCRMR/ppdm/Place-of-manufacture/#4.221.9.1)).
 
-<a name="4.226.4.2">4.226.4.2</a> If the place name as transcribed is known to be fictitious or incorrect, make a note on manufacture giving the actual place name. Include the source of the actual place name (see [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/), [4.221.5.1](/DCRMR/ppdm/Place-of-manufacture/#4.221.5.1)).
+<a name="4.226.4.2">4.226.4.2</a> Make an explanatory note if the [Place of manufacture](/DCRMR/ppdm/Place-of-manufacture/) appearing on the manifestation is known to be fictitious or incorrect (see [4.221.96.1](/DCRMR/ppdm/Place-of-manufacture/#4.221.96.1)).
```

Comment:
Should be "Make a note on manufacture statement giving the basis for the correction if the ..." (continue from [Place of manufacture] unchanged) to mirror instructions at 4.206.4.2.